services/api/index.js

```diff
@@ -1,6 +1,11 @@
 exports.register = function api(server, options, next) {
-  server.route(require('./routes'));
-  next();
+  server.register(require('./lib/postgre'), function(err) {
+    if ( err ) {
+      return next(err);
```

Comment:
Little hard to tell here, but will we throw if pg is unable to connect? If not, any reason why we wouldn't want the API to throw an exception on something as catastrophic as DB connectivity?